docker/s6/services/lazyraster.svc/run

```diff
@@ -1,4 +1,24 @@
 #!/bin/sh
 
+if [ -z "${RASTER_BASE_DIR}" ]; then
+    echo "Failed to start lazyraster: RASTER_BASE_DIR not set"
+    return 1
+fi
+
+echo "Trying to clear existing Lazyraster cached files (if any) in the background..."
+if [ -d "${RASTER_BASE_DIR}" ]; then
+	( \
+		NOW=$(date +%s)
```

Comment:
I think this should be before the sub-shell to prevent a race with starting the service. Putting it outside the sub-shell guarantees that this timestamp is from before the service started.